unstoppable-ios-app/domains-manager-ios/Modules/DomainProfile/Cells/ManageDomainRecordCell/ManageDomainRecordCell.swift

```diff
@@ -209,12 +209,11 @@ private extension ManageDomainRecordCell {
         case .edit(let callback):
             return UIAction(title: action.title, image: action.icon, identifier: .init(UUID().uuidString), handler: { _ in callback() })
         case .editForAllChains(let chains, let callback):
-            if #available(iOS 15.0, *) {
-                let subtitle = chains.prefix(3).map({ $0 }).joined(separator: ", ")
-                return UIAction(title: action.title, subtitle: subtitle, image: action.icon, identifier: .init(UUID().uuidString), handler: { _ in callback() })
-            } else {
-                return UIAction(title: action.title,  image: action.icon, identifier: .init(UUID().uuidString), handler: { _ in callback() })
-            }
+            let subtitle = chains.prefix(3).map({ $0 }).joined(separator: ", ")
```

Comment:
`chains.prefix(3)` is quite specific operation, let's extract into a separate computed var

Reply:
Checked the logic. This is the only place (subtitle of ui action row) where we should shrink chain's abbreviation to 3 chars. Don't think it deserves dedicated var in extension that will be visible everywhere. But i will create a dedicated private function in the cell with clear name 👍